models/articleModels.js

```diff
@@ -52,3 +52,37 @@ exports.updatedVote = (article_id, inc_votes) => {
       });
   }
 };
+
+exports.listOfArticles = (topic) => {
+  const topicNames = ["cats", "mitch", undefined];

```

Comment:
Nice whitelisting but I'd avoid having the undefined in here. Refactor your conditional logic